frontend/components/Pages/HelpAndSupportPage.tsx

```diff
@@ -54,53 +45,69 @@ export const HelpAndSupport = () => {
     getTag();
   }, [getAppVersion, isMounted]);
 
+  const helpItems: HelpItem[] = useMemo(
+    () =>
+      compact([
+        latestTag
+          ? {
+              label: 'Latest release notes',
+              href: `${GITHUB_API_RELEASES}/tag/v${latestTag}`,
+              isExternal: true,
+            }
+          : null,
+        {
+          label: 'Olas community Discord server',
+          href: SUPPORT_URL,
+          isExternal: true,
+        },
+        {
+          label: 'Frequently asked questions',
+          href: FAQ_URL,
+          isExternal: false,
+        },
+        {
+          label: 'Pearl Terms and Conditions',
+          href: TERMS_AND_CONDITIONS_URL,
+          isExternal: false,
+        },
+      ]),
+    [latestTag],
+  );
+
   return (
-    <Card
-      title={<SettingsTitle />}
-      bordered={false}
-      styles={{ body: { paddingTop: 0, paddingBottom: 0 } }}
-      extra={isUserLoggedIn ? <GoToMainPageButton /> : <GoToLoginPageButton />}
-    >
-      <CardSection $borderBottom $padding="16px 24px 24px" vertical>
-        <Title level={5} className="m-0 mb-16 text-base">
-          Frequently asked questions
-        </Title>
-        <a target="_blank" href={FAQ_URL} className="mb-8">
-          Read FAQ {UNICODE_SYMBOLS.EXTERNAL_LINK}
-        </a>
-        {latestTag && (
-          <a
-            href={`${GITHUB_API_RELEASES}/tag/v${latestTag}`}
-            target="_blank"
-            rel="noopener noreferrer"
-            className="mb-8"
+    <Flex className="help-and-settings-container">
+      <Title level={3}>Help Center</Title>
+      <Card styles={{ body: { paddingTop: 8, paddingBottom: 8 } }}>
+        {helpItems.map(({ label, href, isExternal }, index) => (
+          <CardSection
+            key={index}
+            $borderBottom={index !== helpItems.length - 1}
+            $padding="16px"
+            vertical
           >
-            Release notes {UNICODE_SYMBOLS.EXTERNAL_LINK}
-          </a>
-        )}
-        <a target="_blank" href={TERMS_AND_CONDITIONS_URL}>
-          Terms and Conditions {UNICODE_SYMBOLS.EXTERNAL_LINK}
-        </a>
-      </CardSection>
+            <a href={href} target="_blank" rel="noopener noreferrer">
+              <Flex justify="space-between" align="center">
+                {label}
 
-      <CardSection $borderBottom $padding="16px 24px 24px" vertical>
-        <Title level={5} className="m-0 mb-8 text-base">
-          Ask for help
-        </Title>
-        <Paragraph type="secondary" className="mb-16 text-sm">
-          Get your questions answered by the community.
-        </Paragraph>
-        <a target="_blank" href={SUPPORT_URL}>
-          Olas community Discord server {UNICODE_SYMBOLS.EXTERNAL_LINK}
-        </a>
-      </CardSection>
+                {isExternal ? (
+                  <ExternalLink color={COLOR.PURPLE} />
+                ) : (
+                  <ArrowUpRightSvg fill={COLOR.PURPLE} />
```

Comment:
can we please pass the same `fill` prop to both icons? so that we use one approach to change icon color